docs/how-to/cli/addon/addon.md

```diff
@@ -35,6 +45,18 @@ I0111 21:45:25.660129   89345 apply.go:106] "creating object" name="component-ui
 Addon: fluxcd enabled Successfully.
 ```
 
+You can choose one special version of this addon by add `--version` flag in this command. eg:
+
+```shell
+$ vela addon enable fluxcd --version=1.0.0
```

Comment:
```suggestion
vela addon enable fluxcd --version=1.0.0
```

If we don't show the output info, we can omit the `$`, so user can copy paste the command easily.